### PR TITLE
New pull request after "refresh" update

### PR DESCRIPTION
**Pull request by ivanaka**

### CODE REVIEW

**Comment by 0xclem** [comment]
Thanks @ivanaka! Can you please put your translation into the `translation.json` file and remove `translation_old.json`?

**Comment by ivanaka** [comment]
Not good in Git, so probably have pulled incomplete translation. I'll fix it shortly!

**Comment by ivanaka** [comment]
Have pulled final translation. Please check and review. @clementbalestrat

File: synthetix-mintr/ru/translation.json [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+    "gas": "Комиссионные сети Ethereum (Gas):",
+    "tradingFee": "Комиссионные обмена Синтов:",
+    "speed": "Ориентировочная скорость транзакции:"
+  },
+  "maintenance": {
+    "title": "Mintr обновляется и недоступен в данный момент.",
+    "subtitle": "Приносим извинения за неудобства, скоро все наладится."
+  },
+  "mobileLanding": {
+    "title": "Mintr is currently unavailable for mobile and tablet.",
```
Comment (0xclem) [inline synthetix-mintr/ru/translation.json]: @ivanaka you forgot some translations here ;)

**Review by 0xclem — CHANGES_REQUESTED** [review]

File: synthetix-mintr/ru/translation.json (hunk 1 above)
Comment (ivanaka) [reply synthetix-mintr/ru/translation.json]: Fixed missed entries

**Review by 0xclem — APPROVED** [review]

**Comment by 0xclem** [comment]
Looks good, thanks @ivanaka! Will merge it now, review the translation and will keep you updated.

**Pull request merged**